### PR TITLE
Allow absolute imports from src directory

### PR DESCRIPTION
**Pull request by born2bewild**
resolves GH-38, should be merged after GH-41

### CODE REVIEW

File: src/components/Arronax.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import * as React from 'react';
 import { TezosFilter } from 'conseiljs';
-import { DataView } from '../types';
-import { FilterPanel } from './FilterPanel';
-import { DataPanel } from './DataPanel';
-import { NetworkSelector } from './NetworkSelector';
+import { DataView } from 'types';
+import { FilterPanel } from 'components/FilterPanel';
+import { DataPanel } from 'components/DataPanel';
+import { NetworkSelector } from 'components//NetworkSelector';
```
Comment (developer0623) [inline src/components/Arronax.tsx]: Please remove a **/**

File: src/components/DataPanel.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import { DataView } from '../types';
 import { TezosFilter } from 'conseiljs';
-import { Tab } from './Tab';
-import '../style.css';
+import { DataView } from 'types';
+import { Tab } from 'components//Tab';
```
Comment (developer0623) [inline src/components/DataPanel.tsx]: Please remove a /

**Comment by vishakh** [comment]
Could you please explain why this is better than what we currently have?

**Comment by born2bewild** [comment]
lets assume that we have directory structure like :
- components
-- ExpandedList
--- index.tsx
- containers
-- Blocks
--- index.tsx

in containers/Blocks/index.tsx, you want to import ExpandedList component,
currently import looks like `import ExpandedList from '../../components/ExpandedList';` after merging this PR it should be `import ExpandedList from 'components/ExpandedList';`

**Comment by umurb** [comment]
@born2bewild can you please resolve the conflict and take a look at this error so that I can merge this PR;

screen shot 2018-10-23 at 6 06 50 pm

**Comment by born2bewild** [comment]
@umurb fixed

**Pull request merged**